install/index.adoc

```diff
@@ -178,6 +178,18 @@ one HAProxy load balancer, and two
 xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[nodes]
 using the `native` HA method. etcd runs as static pods on the master nodes:
 
+[IMPORTANT]
+====
+Routers and master nodes must be load balanced to have a highly available and
+fault-tolerant environment. Red Hat recommends the use of an enterprise-grade
+external load balancer for production environments. This Load balancing applies
+to the masters, nodes, and hosts running the {product-title} routers.
+Transmission Control Protocol (TCP) layer 3/4 load balancing, in which the load
+is spread across IP addresses, is recommended. See
+link:http://v1.uncontained.io/playbooks/installation/load_balancing.html[External
+Load Balancer Integrations with OpenShift Enterprise 3] for a reference design.
```

Comment:
We may want a note with this that (we don't recommend this 'example' for production use-cases; as it itself is a single point of failure (SOPF), and it's only when you pair this with information from: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/load_balancer_administration/index#s2-lvs-keepalived-haproxy-VSA can you get an HA deployment of Haproxy (from Red Hat) that creates a robust enterprise grade load balancer.